cognite/seismic/protos/query_service.proto

```diff
@@ -113,6 +113,11 @@ service Query {
     Returns the trace in a file that is closest to a point given its coordinates (x,y)
     **/
     rpc GetTraceByCoordinates (CoordinateTraceQueryRequest) returns (Trace) {}
+    /**
+    Returns a trace for each coordinate (inline, xline) from the input stream
+    **/
+    rpc GetTracesByPosition(stream LineTraceQueryRequest) returns (stream Trace) {}
```

Comment:
I think we can go farther and remove the GetTraceByLine (single trace). Nobody is using it currently, and then, if someone wants to request a single trace, they would just send a request to this new endpoint and specify only one trace in the list. Does it make sense?

Also (now just a matter of personal taste), maybe we could name it GetTracesByGridPosition or GetTracesByLines, to avoid confusion of what is a position and what is a coordinate? I am fully aware we called it PositionQuery before, and that feels weird as well. ByLines goes with the same logic of the other methods (GetSlice and GetCube), but if you have a good suggestion, we could change those as well. Naming *is* hard :)

Reply:
done